src/util.js

```diff
@@ -649,6 +649,12 @@ export const getContexts = (key, data=store.getState().data) => {
     // console.error(`getContexts: Unknown key "${key}" context: ${items.join(',')}`)
     return []
   }
+
+  // return when the key is --- because divider should not have superscript
+  if (key === '---') {
+    return []
+  }
+
```

Comment:
This should also happen "higher up" in order to keep functions solely focused on their purpose. i.e. `getContexts` should not have to know anything about specific thoughts. Instead, make the check in `<Superscript />`. I believe there is already a check to prevent empty thoughts from rendering a superscript. Ideally you should add to that conditional.